src/components/ui/VideoGrid/Styled.tsx

```diff
@@ -218,6 +218,134 @@ const ratioStyles = {
+      'ft ft ft ft ft ft v' 'ft ft ft ft ft ft v' 'ft ft ft ft ft ft x';`,
+
+  '23': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+  '23.slim': 'grid-template: repeat(8,1fr) / repeat(3,1fr);',
+  '23.r1by2': 'grid-template: repeat(6,1fr) / repeat(4,1fr);',
+  '23.r1by1': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+
+  '23.featured': `grid-template: repeat(11,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft';`,
+  '23.r1by1.featured': `grid-template: repeat(8,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v'  
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft x';`,
+
+  '24': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+  '24.slim': 'grid-template: repeat(8,1fr) / repeat(3,1fr);',
+  '24.r1by2': 'grid-template: repeat(6,1fr) / repeat(4,1fr);',
+  '24.r1by1': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+
+  '24.featured': `grid-template: repeat(11,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft';`,
+  '24.r1by1.featured': `grid-template: repeat(9,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft x';`,
+
+  '25': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+  '25.slim': 'grid-template: repeat(9,1fr) / repeat(3,1fr);',
+  '25.r1by2': 'grid-template: repeat(7,1fr) / repeat(4,1fr);',
+  '25.r1by1': 'grid-template: repeat(5,1fr) / repeat(5,1fr);',
+
+  '25.featured': `grid-template: repeat(11,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 
+      'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft' 'ft ft ft ft ft ft ft ft';`,
+  '25.r1by1.featured': `grid-template: repeat(10,1fr) / repeat(8,1fr);
+    grid-template-areas:
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 
+      'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft v' 'ft ft ft ft ft ft ft x';`,
+
+  '26': 'grid-template: repeat(6,1fr) / repeat(5,1fr);',
+  '26.slim': 'grid-template: repeat(9,1fr) / repeat(3,1fr);',
+  '26.r1by2': 'grid-template: repeat(7,1fr) / repeat(4,1fr);',
+  '26.r1by1': 'grid-template: repeat(6,1fr) / repeat(5,1fr);',
+
+  '26.featured': `grid-template: repeat(11,1fr) / repeat(9,1fr);
```

Comment:
I wonder whether we should just fix to only 9 row and 9 columns max and switch to 2 rows for remaining videos. The trade off is that the feature tile will be smaller but I think it will be fine.

Reply:
I don't quite get this idea, as if 9 column max and 2 rows for remaining videos, we only support 2*9=18 remaining videos then.

Reply:
We can do 3 row instead of 2 (so only 6 for feature tile)?

Reply:
Ok, I can change 20.featured, 21.featured, ..., 26.featured to use 9*9